packages/oscal-react-library/src/components/OSCALCatalogGroups.tsx

```diff
@@ -7,32 +7,39 @@ import { styled } from "@mui/material/styles";
 import Tab from "@mui/material/Tab";
 import Tabs from "@mui/material/Tabs";
 import PropTypes from "prop-types";
-import React, { useEffect } from "react";
+import React, { ReactNode, useEffect } from "react";
 import { OSCALSection, OSCALSectionHeader } from "../styles/CommonPageStyles";
 import OSCALCatalogGroup from "./OSCALCatalogGroup";
 import OSCALControlParamLegend from "./OSCALControlParamLegend";
 import OSCALAnchorLinkHeader from "./OSCALAnchorLinkHeader";
 import { conformLinkIdText } from "./oscal-utils/OSCALLinkUtils";
+import { OSCALMarkupLine } from "./OSCALMarkupProse";
+import { Control, ControlGroup } from "@easydynamics/oscal-types";
 
 export const OSCALControlList = styled(List)`
   padding-left: 2em;
   padding-right: 2em;
 `;
 
-function TabPanel(props) {
-  const { children, groupId, value, ...other } = props;
+interface TabPanelProps {
+  children: ReactNode;
+  groupId: string;
+  value: string;
+}
+
+const TabPanel: React.FC<TabPanelProps> = (props) => {
+  const { children, groupId, value } = props;
```

Comment:
```suggestion
const TabPanel: React.FC<TabPanelProps> = ({ children, groupId, value }) => {
```

Just so you're aware, you can destructure `props` in the parameter list if you want. At some point, we should pick one or the other to do.